cargo-concordium/src/build.rs

```diff
@@ -314,6 +314,45 @@ pub fn build_contract_schema<A>(
     Ok(schema)
 }
 
+/// Create a new Concordium smart contract project from a template, or there
+/// are runtime exceptions that are not expected then this function returns
+/// Err(...).
+///
+/// Otherwise a boolean is returned, signifying that the creation was
+/// successful.
```

Comment:
It looks like this always returns either an error or `Ok(true)`. If that is the case we should remove the boolean and just return `Ok(())`.